vivo-project/VIVO
# Sprint i18n with main

# What does this pull request do?
This pull-request merges all of the `sprint-i18n` work back into the `main` branch.
In order to retain the history of the `sprint-i18n` effort, **this pull-request should not be squashed**.

Note: Once https://github.com/vivo-project/VIVO-languages/pull/94 and https://github.com/vivo-project/Vitro-languages/pull/43 have been merged and deployed to the Sonatype snapshot repository, this PR depends on a local build of VIVO-languages and Vitro-languages.

# What's new?
The primary update with this PR is the native availability of VIVO-supported languages without the need for any "pom.xml" edits.
The languages files for the languages found in [VIVO-languages](https://github.com/vivo-project/VIVO-languages) are now included in the VIVO war file.

# How should this be tested?
The full extent of testing this PR will not be detailed here; however, two sanity tests should be performed:
1. Build VIVO:
   ```
   mvn clean install -Dvivo-dir=your-vivo-home
   ```
1. Deploy with no i18n configured in `runtime.properties`
1. Deploy with i18n configured in `runtime.properties`

# Interested parties
Tag (@ mention) interested parties or, if unsure, @VIVO-project/vivo-committers
